modules/nodes-pods-vertical-autoscaler-uninstall.adoc

```diff
@@ -27,3 +29,105 @@ You can remove a specific VPA using the `oc delete vpa <vpa-name>` command. The
 . Optional: To remove all operands associated with the Operator, in the dialog box, select *Delete all operand instances for this operator* checkbox.
 
 . Click *Uninstall*.
+
+. Optional: Use the {product-title} CLI to remove the VPA components:
```

Comment:
I've generally seen us avoiding the attribute and writing as "OpenShift CLI", such as here: https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html